docs/users.rst

```diff
@@ -26,20 +41,157 @@ image owners can use ``koji watch-logs <task id> | grep -w x86_64``.
 Configuring koji-containerbuild for koji CLI
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+To be able to build container images from CLI `koji-containerbuild plugin`_
+must be installed.
+
+To configure CLI you'll need to copy [koji] section in /etc/koji.conf to
```

Comment:
mmm I am not sure if that is true

basically when you have koji/brew installed you have to install just koji_containerbuild, and IF brew/koji is able to import that module it will also support container-build option

Reply:
Robert's right, I don't think you need to update koji.conf.

Reply:
In that case, also koji-contianerbuild docs are outdated: https://github.com/containerbuildsystem/koji-containerbuild#koji-cli

Reply:
yes, I don't remember if that was ever true